crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -24,12 +24,22 @@ struct Metrics {
     native_price_cache_background_updates: IntCounter,
     /// number of items in cache that are outdated
     native_price_cache_outdated_entries: IntGauge,
+    /// Tracks the CoinGecko batch size
+    #[metric(labels("result"))]
+    coin_gecko_batch_size: IntCounterVec,
```

Comment:
I assume, this metric shouldn't contain any labels.
```suggestion
    coin_gecko_batch_size: IntCounter,
```